test/unit/app/browser/api/ledgerTest.js

```diff
@@ -3753,6 +3753,24 @@ describe('ledger api unit tests', function () {
     })
   })
 
+  describe('disablePayments', function () {
+    beforeEach(function () {
+      onChangeSettingSpy.reset()
+    })
+
+    it('calls changeSetting', function () {
+      ledgerApi.disablePayments()
+      assert(onChangeSettingSpy.calledOnce)
+    })
+
+    it('goes to set PAYMENTS_ENABLED to false', function () {
+      ledgerApi.disablePayments()
+      const args = onChangeSettingSpy.getCall(0).args
```

Comment:
nit: this can be one liner `assert(onChangeSettingSpy.withArgs(settings.PAYMENTS_ENABLED, false).calledOnce)`